.ideainspect

```diff
@@ -21,3 +15,10 @@ resultdir: target/inspection-results
 
 # IDEA project root directory containing the ".idea" directory
 # rootdir: .
+
+# Library folder containing idea.properties More info at:
+# http://tools.android.com/tech-docs/configuration
+# lib: ~/Library/Preferences/AndroidStudio2.1/
```

Comment:
One of the things that we need to access is idea.properties, but not the idea.properties in the bin path. The link provided shows you where the "personalized" overrides of idea.properties are, so that we can work with global installations while accessing local settings.